app/views/idv/shared/other_failure.html.slim

```diff
@@ -0,0 +1,5 @@
+= render 'shared/failure', presenter: presenter
```

Comment:
One change we want to make before merging is rename this `idv_failure` and move it into the `idv` folder